### PR TITLE
Fix android web socket error

### PR DESCRIPTION
**Pull request by csduarte**
#### Summary
Fixes the android websocket error where the wrong origin was being sent.

@jarredwitt

### CODE REVIEW

File: service/client/websocket_client.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ class WebSocketClient {
         this.stop = false;
     }
 
-    initialize(forceConnection = true, connectionUrl = this.connectionUrl, token, dispatch, getState, webSocketConnector = WebSocket) {
+    initialize(forceConnection = true, connectionUrl = this.connectionUrl, token, dispatch, getState, webSocketConnector = WebSocket, platform) {
```
Comment (hmhealey) [inline service/client/websocket_client.js]: I wonder if we should wrap these arguments in an object. Thoughts?

Comment (enahum) [inline service/client/websocket_client.js]: Yes, I would agree with that actually

Comment (jarredwitt) [inline service/client/websocket_client.js]: I'll go ahead and add in that change.

**Comment by jarredwitt** [comment]
@hmhealey @enahum take a look now. Thought we should leave ```authToken, dispatch, and getState``` since those don't have defaults set. Open to suggestions.

**Comment by hmhealey** [comment]
Yeah, it makes sense to leave them there for now

When we start trying to share this code between the web app and RN that we should remove those fields from the websocket client entirely. We're shipping the client and websocket client on NPM separately so other people can use them to integrate, so it doesn't make sense for the websocket client to have redux built into it

**Pull request merged**